charts/harvester-cloud-provider/dependency_charts/kube-vip/templates/_helpers.tpl

```diff
@@ -63,12 +63,18 @@ Create the name of the service account to use
 {{- end }}
 
 {{/*
-Global system default registry
+Convert string to boolean
 */}}
-{{- define "system_default_registry" -}}
-{{- if .Values.global.cattle.systemDefaultRegistry -}}
-{{- printf "%s/" .Values.global.cattle.systemDefaultRegistry -}}
```

Comment:
This has broken support for system default registry, and will need to be fixed before this update can go into Rancher or RKE2.

Reply:
Thanks @brandond. 
We are working on https://github.com/harvester/charts/pull/322 for this issue.

cc @FrankYang0529, @bk201